source/_includes/common-tasks/file_access.md

```diff
@@ -19,19 +19,34 @@ Using any of the add-ons listed below, the following directories are made availa
 
 The Samba add-on creates smb shares which can be accessed from another computer. You can also edit files using the editor of your preference from your client computer. This add-on can be installed from the add-on store's official repository.
```

Comment:
> Samba add-on creates smb shares

Cool! what are those?

I think "smb shares" should be a term here 😄

Reply:
Excellent point. Thank you for calling me out on it. Will address this in a separate PR.

Reply:
#32913 
removed the mention of the protocol. I think it should be possible to use the add-on without knowing about the underlying protocol.